src/Mafin.Web.Api.Rest/MafinHttpClient.cs

```diff
@@ -280,4 +280,15 @@ public MafinHttpClient(HttpMessageHandler handler, bool disposeHandler)
 #pragma warning restore CA1054
 
     private static CancellationToken ResolveToken(CancellationToken? token) => token ?? CancellationToken.None;
+
+    // Added for interoperability between targets
+    private async Task<HttpResponseMessage> GetCorrectPatchImplementationResponse(Uri? requestUri, HttpContent? content, CancellationToken cancellationToken)
+    {
+#if NETSTANDARD2_1_OR_GREATER
+        var response = await PatchAsync(requestUri, content, cancellationToken).ConfigureAwait(false);
+#else
+        var response = await SendAsync(new HttpRequestMessage(new HttpMethod("PATCH"), requestUri) { Content = content }, cancellationToken).ConfigureAwait(false);
```

Comment:
`SendAsync` method can be internally used by all higher methods (like `GetAsync`). There is no need to introduce strange `GetCorrectPatchImplementationResponse` name.

Reply:
I'd prefer not to use SendAsync everywhere directly, doesn't make much sense, just more noise in my opinion. Anyways, with target changed to `netstandard2.0` both the code and the name of the function are shorter and easier on eye.